src/components/huddle.tsx

```
@@ -0,0 +1,83 @@
+  };
+
+  const declineCall = () => {
+    call?.leave({
+      reject: true,
+    });
+  };
+
+  if (!call) return null;
+
+  switch (true) {
+    case callingState === CallingState.RINGING && call.isCreatedByMe && call.id === channel.id:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)